models/mart/average_revenue.sql

```diff
@@ -0,0 +1,44 @@
+with customer_count as 
+	(select 

```

Comment:
Should we keep the same style, i.e.:

```suggestion
with customer_count as (
	select 
```

WDYT?